app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+  def index
+    # require "pry"; binding.pry
+    # @recruiter = current_user
```

Comment:
@jimszalew 
Is our plan to move use "current_user" dynamically for all user types in one dashboard?

Reply:
Yeah, I forgot to delete the commented out section of the dashboard controller here. I was thinking that the dashboard index view would have one 'render partial' with something like 'current_user.type' statement that renders a different partial based on what type of user 'current_user' is. Each partial for each type of user could still call 'current_user'.

Reply:
I LIKE IT! 

:taco: for @jimszalew